x-pack/plugins/fleet/server/services/elastic_agent_manifest.ts

```diff
@@ -353,12 +368,19 @@ spec:
                   fieldPath: metadata.name
           securityContext:
             runAsUser: 0
+            capabilities:
+              add:
+              # The following capabilities are needed for 'Defend for containers' integration (cloud-defend)
+              # If you are not using this integration, then these capabilites can be removed.
+                - BPF # (since Linux 5.8) allows loading of BPF programs, create most map types, load BTF, iterate programs and maps.
+                - PERFMON # (since Linux 5.8) allows attaching of BPF programs used for performance metrics and observability operations.
+                - SYS_RESOURCE # Allow use of special resources or raising of resource limits. Used by `Defend for Containers` to modify `rlimit_memlock`
           resources:
             limits:
-              memory: 500Mi
+              memory: 700Mi
```

Comment:
This memory settings and the next one at line 361 are the only bits we are aware of at cloud-native-monitoring team. As far as I know